docs/guides/process-collector.md

```diff
@@ -0,0 +1,84 @@
+---
+title: 'The "process-collector" gadget'
+weight: 10
+---
+
+The process-collector gadget gathers information about running processes.
+
+Let's start this demo by creating a namespace:
+
+```
+$ kubectl create ns demo
+namespace/demo created
+```
+
+There is not any running process in the `demo` namespace now:
+
+```
+$ kubectl gadget process-collector -n demo
+NAMESPACE    POD    CONTAINER    COMM    PID
+```
+
+Create a pod on the `demo` namespace:
+
+```
+$ kubectl -n demo run mypod --image=ubuntu -- sleep 100000
+pod/mypod created
+```
+
+After the pod is running, we can try to get the list of running processes again:
+
+```
+$ kubectl gadget process-collector -n demo
+NAMESPACE    POD      CONTAINER    COMM     PID
+demo         mypod    mypod        sleep    327960
+```
+
+We can see the `sleep` command that we used when creating `mypod`.
+
+Execute a shell on the pod:
+
+```
+$ kubectl -n demo exec -it mypod -- /bin/sh
+#
+```
+
+Now there are two processes running in `mypod`:
+
+```
+NAMESPACE    POD      CONTAINER    COMM     PID
+demo         mypod    mypod        sh       327960
+demo         mypod    mypod        sleep    327785
+```
```

Comment:
Why you didn't add `$ kubectl gadget process-collector -n demo` here? Just to not be redundant?

Reply:
I think it is better to be redundant for clarity

Reply:
Bad copy & paste.